lib/honeybadger/client.ex

```diff
@@ -8,7 +8,7 @@ defmodule Honeybadger.Client do
   @headers [
     {"Accept", "application/json"},
     {"Content-Type", "application/json"},
-    {"User-Agent", "Honeybadger Elixir"}
+    {"User-Agent", "Honeybadger Elixir #{Honeybadger.Mixfile.project()[:version]}"}
```

Comment:
This won't change at runtime. Using a module attribute would avoid the extra function call on every invocation.

Reply:
Using module attribute that means we will have to edit this module each time the version is changed.

Reply:
Not if you use `@external_attribute` for the mix file. Then it will track changes and force recompilatiin automatically.

Reply:
Hello @sorentwo , I haven't come across `@external_attribute`  before . I was looking into the docs and came across this https://hexdocs.pm/elixir/Module.html#module-external_resource. Is it the same thing?

Reply:
Yes, that’s the same thing. It forces a recompile when an external file changes.

Reply:
Hello @sorentwo , I have made the change . Let me know if thats how it should be